modules/caddypki/pki.go

```diff
@@ -54,15 +54,9 @@ func (p *PKI) Provision(ctx caddy.Context) error {
 	p.ctx = ctx
 	p.log = ctx.Logger(p)
 
-	// if this app is initialized at all, ensure there's at
-	// least a default CA that can be used: the standard CA
-	// which is used implicitly for signing local-use certs
 	if p.CAs == nil {
 		p.CAs = make(map[string]*CA)
 	}
-	if _, ok := p.CAs[DefaultCAID]; !ok {
```

Comment:
Can probably remove the lines above it too (L60-62), since ProvisionDefaultCA() makes sure the map isn't nil.

Reply:
Doesn't it need to be non-nil to `range p.CAs`?

Reply:
Nope, reads on nil maps are safe.